src/python/pants/backend/python/typecheck/pytype/BUILD

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+resource(name="lockfile", source="pytype.lock")
+
+python_sources(
+    overrides={"subsystem.py": {"dependencies": [":lockfile"]}},
+)
+
+python_tests(
+    name="tests",
+    overrides={
+        "rules_integration_test.py": {"timeout": 240},
```

Comment:
If this timeout is merely copied over from pyright, perhaps drop it until we see it is in fact needed? (or start with a lower timeout)

Reply:
dropped